src/Resources/Locales/en.json

```diff
@@ -625,7 +625,7 @@
         "powerBI": {
             "title": "Power BI widget",
             "libraryName": "Power BI Visual",
-            "description": "Render a Power BI visual or tile",
+            "description": "Embed a Power BI visual or tile",
```

Comment:
make sure you bump the loc files by changing the key of this string and push to remote. once all the files have the new key, revert it so they all transalte this update.

Reply:
you can just change teh parent key as well and it will regen all teh children.

Reply:
But the higher you go, the less bonus you get at the end of the year. ;)

Reply:
Straight to the root!!! 🚀

Reply:
Thanks there were some lorem ipsum in there from my initial commit when I just needed *something*.